resources/analysisTools/indelCallingWorkflow/TiN_CanopyBasedClustering.R

```diff
@@ -231,7 +241,8 @@ if("Somatic_Rescue" %in% rescueInfo.toFile$TiN_Class) {
   rescueInfo.toFile<-rbind(rescueInfo.toFile, c("Somatic_Rescue", 0, 0, 0))
   rescueInfo.toFile$Pid<-opt$pid
 }
-write_json(rescueInfo.toFile, path=paste0(opt$oFile, "_summary.json"))
```

Comment:
Is there an equivalent file which replaces the removed file?

Reply:
Yes, checkSampleSwap.json will contain this information.